src/pal/src/eventprovider/lttng/tpdotnetruntimeprivate.h

```diff
@@ -0,0 +1,1600 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Do not modify the file. This file is generated using the script from <root>/src/inc/genXplatEtw.pl
+//
+
+#include "palrt.h"
+#include "pal.h"
+
+#undef TRACEPOINT_PROVIDER
+#define TRACEPOINT_PROVIDER dotnetruntimeprivate
```

Comment:
Stylistic: Can we match the  provider names that we use with ETW, including casing:  DotNETRuntime_Private, etc.